welcome/src/main/kotlin/edu/artic/welcome/WelcomeFragment.kt

```diff
@@ -35,5 +50,61 @@ class WelcomeFragment : BaseViewModelFragment<WelcomeViewModel>() {
             }
         }
 
+        val list = getTours()
+
+        context?.let {
+            tourSummaryRecyclerView.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
+            tourSummaryRecyclerView.adapter = ToursAdapter(list, it)
+            welcomePreferencesManager = WelcomePreferencesManager(it)
+        }
+
+        /**
+         * Peak Animation.
+         * Scroll RecyclerView to the last item and back again to first item.
+         */
+        val peekedTourSummary = welcomePreferencesManager?.peekedTourSummary ?: false
+
+        if (!peekedTourSummary) {
+            handler.postDelayed({
```

Comment:
We should use RX if possible. and handle subscriptions and cancel them if theyre running when view destroyed.